internal/consensus/metrics.go

```diff
@@ -44,7 +44,7 @@ type Metrics struct {
 	// Number of validators who did not sign.
 	MissingValidators metrics.Gauge
 	// Total power of the missing validators.
-	MissingValidatorsPower metrics.Gauge
+	MissingValidatorsPower metrics.Gauge `metrics_labels:"validator_address"`
```

Comment:
This fails right now 

https://github.com/sei-protocol/sei-tendermint/blob/1e6789900d665f0b8509e94f1243341ba47e7f31/internal/consensus/state.go#L2254-L2258

Reply:
nice catch